frontend/src/model/beta/reviews/ReviewBanner.tsx

```diff
@@ -19,13 +19,15 @@ type ReviewBannerProps =
   | {
       release: ReleaseInterface
       accessRequest?: never
+      removeBorderRadius?: boolean
     }
   | {
       release?: never
       accessRequest: AccessRequestInterface
+      removeBorderRadius?: boolean
```

Comment:
Not 100% on the name here, how about `square`? Also you don't need to add this new prop twice. You can do the following:
```
type ReleaseOrAccessRequest =
  | {
      release: ReleaseInterface
      accessRequest?: never
    }
  | {
      release?: never
      accessRequest: AccessRequestInterface
    }

type ReviewBannerProps = {
  square?: boolean
} & ReleaseOrAccessRequest
```